doc/changelog.md

```diff
@@ -135,6 +135,10 @@
   and remove references to QIR in runtime's Makefile.
   [#243](https://github.com/PennyLaneAI/catalyst/pull/243)
 
+* On gradient tests, do not use finite-diff method for expected values to avoid compatiblity
+  issues with autograd.
+  [#249](https://github.com/PennyLaneAI/catalyst/pull/249)
+

```

Comment:
Since this only affects our test suite which doesn't affect users I don't think this changelog is needed :) 
```suggestion
```

It also looks like the fix doesn't avoid finite diff but wraps to numpy arrays instead 😅

Reply:
Ah yes. I changed my mind after writing the changelog.